charts/optimize-live/values.yaml

```diff
@@ -44,10 +44,17 @@ recommender:
       memory: 100Mi
 
 grafana:
+  enabled: true
   image:
     repository: grafana/grafana-oss
     pullPolicy: IfNotPresent
     tag: 9.1.5
+  componentNameOverride: ""
```

Comment:
nitpick: `nameOverride` is probably good sufficient since everything under `grafana` should apply only to the grafana component